src/main/resources/application.conf

```diff
@@ -6,7 +6,7 @@ centaur {
   # The maximal length of a workflow, intended as a sanity check and not actually a test in and of itself
   maxWorkflowLength: 2 hours
   # Cromwell's metadata is eventually consistent. Set a timeout such that we expect it to have eventually consisted
-  metadataConsistencyTimeout: 10 seconds
+  metadataConsistencyTimeout: 5 minutes
```

Comment:
That'd be good. 

While I'm interested to know the results of this increased time I would be unhappy with us actually giving 5 minutes before failing. That's well beyond what we expect to see so it's likely a problem if we see it.